src/saving/SaveHelper.cs

```diff
@@ -307,7 +307,11 @@ public static (int Count, ulong DiskSpace) CountSaves(Regex? nameMatches = null)
     public static void DeleteSave(string saveName)
     {
         using var directory = new Directory();
-        directory.Remove(Path.Combine(Constants.SAVE_FOLDER, saveName));
+        var finalPath = Path.Combine(Constants.SAVE_FOLDER, saveName);
+        directory.Remove(finalPath);
```

Comment:
Would it be more optimal to handle its return value instead?

Reply:
We aren't deleting that many files, so I preferred the API which is much more likely to detect a problem, after all I still remember that one Godot operation that just always returned Ok on Windows...

Reply:
Either is fine I guess.